server/NewBankClientHandler.java

```diff
@@ -8,56 +8,84 @@
 
 public class NewBankClientHandler extends Thread {
 
-	private NewBank bank;
+	private newbank.server.NewBank bank;
 	private BufferedReader in;
 	private PrintWriter out;
 
 	public NewBankClientHandler(Socket s) throws IOException {
-		bank = NewBank.getBank();
+		bank = newbank.server.NewBank.getBank();
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it